packages/toolpad-app/src/toolpad/propertyControls/SelectOptions.tsx

```diff
@@ -29,6 +29,7 @@ function SelectOptionsPropEditor({
   value = [],
   onChange,
 }: EditorProps<(string | SelectOption)[]>) {
+  const [addOptionState, setAddOptionState] = React.useState(false);
```

Comment:
What if we use this state for any possible error message, so we could call this `[optionErrorMessage, setOptionErrorMessage]`, and it would be a `string` with an error message instead of a boolean? 
That way we can use it for more things later on, if we need to.

Reply:
What message should be set for this?

`Do not input <kbd>same</kbd> value`,
`Do not input <kbd>same</kbd> value`
There are two errors message, with some element marks in the message

Reply:
If you just want to clarify what it means, I think the variable name is also a key point,

Reply:
`optionErrorMessage` would have to be a `string` instead of a `boolean` so that we can set any error message.
"Do not enter duplicate value" could work as an error message!

Reply:
If we need to include tags such as `<kbd>` you can use `React.ReactNode` as the type for the error message, instead of a `string`.